src/client/pages/Offer/Introduction/Sidebar/StartDate.tsx

```diff
@@ -331,8 +319,9 @@ export const StartDate = ({
     quoteIds: Array<string>,
   ) => {
     try {
-      const dateIsTooSoon = isLessThanOneWeekAway(newDateValue)
-      setShowToSoonError(userIsCarSwitcher && dateIsTooSoon)
+      setShowToSoonError(
```

Comment:
It shouldn't happen, but now if `newDateValue` is `null`, this evaluates to `true` and we show the error message. Before, it didn't. 

But it might be an improvement actually since a `null` date definitely would want us to show the warning 💡

Reply:
Fixed this and made check more readable.  If we want warnings for nulls this should be handled elsewhere